index.html

```diff
@@ -995,7 +1118,11 @@ <h3>Script Event</h3>
               However, there can be cases where they do,
               e.g. in <a>Dubbing Scripts</a> when different <a>Characters</a> speak different text at the same time.</p>
           </li>
-          <li>An optional <a>Script Event Type</a> used to identify the type of event (e.g. dialogue, on screen text) in <a>Dubbing Scripts</a>.</li>
+          <li>A <a>Represents</a> property
+            used to identify what content the event represents (e.g. dialogue, on screen text, etc.).
+            Every <a>Script Event</a> MUST have a valid <a>Represents</a> property.
+          <aside class="note">The <a>Represents</a> property is inherited,
```

Comment:
```suggestion
          <aside class="note">The <a>Represents</a> property may be inherited,
```

Reply:
Won't put a normative keyword in a note, but will change "inherited" to "inheritable".

Reply:
Done in 09b070e